### PR TITLE
[翻译完毕] vector

### PR DESCRIPTION
**Pull request by ArkhamWJZ**

### CODE REVIEW

File: locales/zh_CN/LC_MESSAGES/vector.po [hunk 1]
```diff
@@ -3,178 +3,162 @@
...
 
 #: ../../vector.rst:22
 msgid "(DataType) data type of the components"
-msgstr ""
+msgstr "(DataType) 向量中分量的数据类型"
 
 #: ../../vector.rst:23
 msgid "(scalar or tuple) shape the tensor of vectors, see :ref:`tensor`"
-msgstr ""
+msgstr "(scalar or tuple)张量的形状(其中的元素就是向量), 在 :ref:`tensor` 中查看更多细节"
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/vector.po]: I think it's better to match current cn doc status, `gui.po` for example, do translate all 'type descriptors' in `()`. What do you think? Inputs are welcome!
```suggestion
msgstr "(标量或元组) 张量的形状(其中的元素就是向量), 在 :ref:`tensor` 中查看更多细节"
```

File: locales/zh_CN/LC_MESSAGES/vector.po [hunk 2]
```diff
@@ -186,58 +170,52 @@ msgstr ""
...
-msgstr ""
+msgstr "(Vector)向量"
 
 #: ../../vector.rst:109
-msgid ""
-"(optional, scalar) a safe-guard value for ``sqrt``, usually 0. See the "
-"note below."
-msgstr ""
+msgid "(optional, scalar) a safe-guard value for ``sqrt``, usually 0. See the note below."
+msgstr "(optional, scalar) sqrt的安全保护值，通常为0. 详见下面注解。"
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/vector.po:182]: The same to here.
```suggestion
msgstr "(可选，标量) sqrt的安全保护值，通常为0. 详见下面注解。"
```

File: locales/zh_CN/LC_MESSAGES/vector.po [hunk 3]
```diff
@@ -186,58 +170,52 @@ msgstr ""
...
 
 #: ../../vector.rst:129
 msgid "(scalar) the dot (inner) product of ``a`` and ``b``"
-msgstr ""
+msgstr "(scalar) ``a`` 和 ``b`` 之间点乘（内积）的结果"
 
 #: ../../vector.rst:131 ../../vector.rst:159 ../../vector.rst:177
 msgid "E.g., ::"
-msgstr ""
+msgstr "E.g., ::"
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/vector.po]: Not translated? Maybe some cn reader can't understand what `e.g.` mean?

File: locales/zh_CN/LC_MESSAGES/vector.po [hunk 4]
```diff
@@ -3,178 +3,162 @@
...
+msgid "as a temporary local variable. An ``n`` component vector consists of ``n`` scalar values."
+msgstr "作为临时局部变量，一个 ``n`` 分量的向量由标量 ``n`` 组成。"
 
 #: ../../vector.rst:9
-msgid ""
-"as an element of a global tensor. In this case, the tensor is an "
-"N-dimensional array of ``n`` component vectors."
-msgstr ""
+msgid "as an element of a global tensor. In this case, the tensor is an N-dimensional array of ``n`` component vectors."
+msgstr "作为全局张量(global tensor)的构成元素。比如，由向量 ``n`` 组成的N维序列构成了的一个全局张量。"
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/vector.po]: Sorry about not mentioning this in [wiki](https://github.com/taichi-dev/taichi-docs-zh-cn/wiki/%E5%B8%B8%E7%94%A8%E4%B8%93%E6%9C%89%E8%AF%8D%E6%B1%87%E5%AF%B9%E7%85%A7%E8%A1%A8)! Adding this entry now.
```suggestion
msgstr "作为全局张量(global tensor)的构成元素。比如，由向量 ``n`` 组成的N维数组构成了的一个全局张量。"
```

**Review by archibate — COMMENTED** [review]
Nice translation!

File: locales/zh_CN/LC_MESSAGES/vector.po (hunk 3 above)
Comment (ArkhamWJZ) [reply locales/zh_CN/LC_MESSAGES/vector.po]: Sorry for my neglect of double checking for the content.

File: locales/zh_CN/LC_MESSAGES/vector.po (hunk 1 above)
Comment (ArkhamWJZ) [reply locales/zh_CN/LC_MESSAGES/vector.po]: This is a good opinion, I will make an amendment as soon as possible

File: locales/zh_CN/LC_MESSAGES/vector.po (hunk 4 above)
Comment (ArkhamWJZ) [reply locales/zh_CN/LC_MESSAGES/vector.po]: Lol.

**Review by archibate — APPROVED** [review]
Cool! Thank for the hard work :)

**Pull request merged**